### PR TITLE
cahier 0.4.3 (new cask)

### PR DESCRIPTION
**Pull request by chenrui333**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/c/cahier.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  depends_on macos: ">= :big_sur"
+
+  app "Cahier.app"
+
+  zap trash: [
+    "~/Library/Preferences/org.Cahier-App.Cahier.plist",
+    "~/Library/Saved Application State/org.Cahier-App.Cahier.savedState",
+  ]
```
Comment (razvanazamfirei) [inline Casks/c/cahier.rb:22]:
```suggestion
	zap trash: [
	  "~/Documents/Cahier",
	  "~/Library/HTTPStorages/org.Cahier-App.Cahier",
	  "~/Library/Preferences/com.cahier-app.Cahier.plist",
	  "~/Library/Preferences/org.Cahier-App.Cahier.plist",
	  "~/Library/Saved Application State/org.Cahier-App.Cahier.savedState",
	]
```

**Review by razvanazamfirei — COMMENTED** [review]
Does this need `auto_updates true` by any chance?

**Comment by bevanjkay** [comment]
> Does this need `auto_updates true` by any chance?

The presence of sparkle would usually suggest yes.

**Comment by krehel** [comment]
This also needs Rosetta.

cahier

**Comment by krehel** [comment]
Someone should confirm the need for `auto_updates: true` - I'm on an M1 w/o Rosetta (and don't want to install) so can't verify at the moment.

**Comment by chenrui333** [comment]
I did not see the `check for updates...` in the menu bar though

**Comment by bevanjkay** [comment]
Confirmed that it does have an auto-update and install mechanism

**Comment by chenrui333** [comment]
oh, that is interesting. 🤦

**Comment by krehel** [comment]
Added `auto_updates: true` - I think we'll be good here.

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**